app/scripts/modules/netflix/ci/ci.filter.model.ts

```diff
@@ -1,10 +0,0 @@
-import {module} from 'angular';
-
-export class CiFilterModel {
-  public searchFilter: string;
-}
-
-export const CI_FILTER_MODEL = 'spinnaker.netflix.ci.filter.model';
-
-module(CI_FILTER_MODEL, [])
```

Comment:
This change switches from an *instance of CiFilterModel* (as a singleton/angular service) to exporting the *CiFilterModel class*. That means an instance of the  CiFilterModel has to be new'd and managed as a singleton somewhere, right?

I feel that we should formulate our plan for managing global state before changing how singletons  like this one are managed.